pkg/crc/api/types.go

```diff
@@ -47,3 +47,8 @@ type getConfigResult struct {
 	Error   string
 	Configs map[string]interface{}
 }
+
+// startArgs is used to get the pull secret file path as argument for start handler
+type startArgs struct {
+	PullSecretFile string `json:"pullSecretFile"`
```

Comment:
So we are not accepting cpu/mem etc as part of startArgs as of now and either take from config or use defaults?

Reply:
Yeah, right now i only implemented to pass in a pull secret (because if its not set you can't start the cluster) the memory, cpu configs i think should be part of the separate config window, using the config api

Reply:
I am not following why this needs to be an exception. The config could have a text input or file requestor to locate the pull secret? Or does the daemon else not understand this change? Not seeing why this needs to be a start argument.

Reply:
This is only for the startHandler, we want to avoid the manual step of having to configure the `pull-secret-file` via the cli before being able to start the cluster through the api.